doc/posts/zitification/ssh/zitifying-ssh.md

```diff
@@ -0,0 +1,133 @@
+right? (Spoiler alert: wrong)
+
+I'm sure you've guessed that this is not the case whatsoever. After all, attackers don't leave `ssh` alone just because
+it's not worth it to try! Put a machine on the open internet, expose `ssh` on port 22 and watch for yourself all the
+attempts to access `ssh` using known default/weak/bad passwords flood in. Attacks don't only come from the Internet
+either! Attacks from a single compromised machine on your network very well could behave in the same way as an outside
+attacker. This is particularly true for ransomware style attacks as the compromised machine attempts to expand/multiply.
+The problems don't just stop here either. DoS attacks, other zero-day type bugs and more are all waiting for any service
+sitting on the open internet.
+
+A zitified `ssh` client is superior since the port used by `ssh` can be elimitated from the Internet-based firewall or
+perhaps could even be configured to listen only on localhost, preventing any connections whatsoever from any network
+client. In this configuration the `ssh` process is effectively "dark"
+meaning the service is not only invisible to the Internet but also the local area network as well! The only way to
+`ssh` to a machine using a [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network) will
+be to have an identity authorized for
+that [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network). Cool right? Let's see how
+we did that, and how you can do the same thing using
+a [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network).
+
+<hr>
+
+## How It's Done
+
+There are a few steps necessary before being able to use `zssh`:
+
+* Establish a [Ziti Network in AWS](https://github.com/openziti/ziti/blob/release-next/quickstart/aws.md) (using AWS in
+  this example)
+* Create and enroll two Ziti Endpoints (one for our `ssh` server, one for the client)
+    * the `sshd` server will run `ziti-tunnel` for this demonstration. Conviniently it will run on the same machine that
+      I used to setup the [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network).
+    * the client in this case will run `zssh` from my local machine and I'll `zssh` to the other endpoint
+* Create the [Ziti Service](https://openziti.github.io/ziti/services/overview.html) we'll use and authorize the two
+  endpoints to use this service
+* Use the `zssh` binary from the client side and the `ziti-tunnel` binary from the serving side to connect
+* Harden `sshd` further by removing port 22 from any Internet-based firewall configuration (for example from within the
+  security-groups wizard in AWS) or by forcing `sshd` to only listen on `localhost/127.0.0.1`
+
+After preforming these steps you'll have an `sshd` server which is dark to the Internet or your LAN (depending on how
```

Comment:
Misspelled: `performing `

Reply:
Consider `which is` to `that is`

Reply:
> Misspelled: `performing `

gah -i've been 'preforming' things for years! :D